src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactory.cs

```diff
@@ -107,8 +98,8 @@ private sealed class ReflectionAIFunction : AIFunction
     {
         private readonly MethodInfo _method;
         private readonly object? _target;
-        private readonly Func<IReadOnlyDictionary<string, object?>, AIFunctionContext?, object?>[] _parameterMarshalers;
-        private readonly Func<object?, ValueTask<object?>> _returnMarshaler;
+        private readonly Func<IReadOnlyDictionary<string, object?>, AIFunctionContext?, object?>[] _parameterMarshallers;
+        private readonly Func<object?, ValueTask<object?>> _returnMarshaller;
```

Comment:
We can change if you want, but single `l` is the US spelling.

Reply:
Oh ok, I only changed it because my spell checker was complaining.